test_setup.js

```diff
@@ -183,9 +183,10 @@ module.exports = async function (done) {
       process.env[user.envTokenName] = at;
     }
 
+    const mainAcct = users.mainAcct;
     console.log('Delete then create basic client...\n');
     deleteClient('basic-test-client');
-    const basicClient = createClient('basic-test-client', 'test-client@example.com', 'basic');
+    const basicClient = createClient('basic-test-client', mainAcct.username, 'basic');
```

Comment:
was this breaking something?

Reply:
i thought i needed that but i actually don't! haha i'll remove it